<launch>
  <node name="sbg_ellipse" pkg="sbg_driver" type="ellipse">
  	<param name="uart_port" value="/dev/sbg" />
  	<param name="uart_baud_rate" value="115200" type="int" />
  </node>
</launch>
